notes/release.md

```diff
@@ -1,51 +1,52 @@
-# Release (WIP)
+# Release
 
 This note explains how to release deepchem packages.
 
 ## How to release
 
-1. Create and merge a release PR (just modify the version in `deepchem/__init__.py`)
-2. Push a new tag in the merge commit -> release in PyPI
-3. Create and merge a release PR in the [feedstock repository](https://github.com/conda-forge/deepchem-feedstock) -> release in conda forge
-4. Create and merge a PR for updating the Dockerfile (`docker/conda-forge/Dockerfile`)
-5. Build and publish a new docker image -> release in DockerHub 
-6. Fix the version in README.md and Documentation
+1. Create and merge a release PR
+    - Modify the version in `deepchem/__init__.py` (Remove `.dev`)
+    - Update the documents for installing a new package in `README.md` and `docs`
```

Comment:
Additional work.

Previously, I wrote that we modify the docs after pushing a new tag. It is a mistake. 
Considering the docs for 2.4.0 tag, the installation section is still old. https://deepchem.readthedocs.io/en/2.4.0/get_started/installation.html

Reply:
Ah good catch!